docs/rules/default.md

```diff
@@ -1,5 +1,9 @@
 # import/default
 
+💼 This rule is enabled in the following configs: ❗ `errors`, ✅ `recommended`.
```

Comment:
let's use the black and white check here for "recommended" also

Reply:
Updated using `--config-emoji recommended,☑️`.